basis/node/node.py

```diff
@@ -78,6 +88,14 @@ class OutputStream(_OutputMeta, OutputStreamMethods):
     pass
 
 
+class Table(_InputMeta, InputTableMethods, OutputTableMethods):
```

Comment:
These classes are not callable in the node function form, so they don't need metaclasses. Instead, we should just define `__init__` stubs for them. We'll also need to add the `mode` parameter.